lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -362,6 +365,50 @@
                 },
 
                 /**
+                 * Creates a searchable view with a header that understands
+                 * how to handle searches.
+                 */
+                createSearchableListView: function (options) {
+                    var subject = options.subject,
+                        collection = options.collection,
+                        searchFieldView = new SearchFieldView({
+                            collection: collection
+                        }),
+                        view = this.createViewWithHeader({
+                            mainView: options.mainView,
+                            subject: options.subject,
+                            headerType: 'search',
+                            headerActionsView: searchFieldView
+                        }),
+                        headerModel;
+                    headerModel = view.header.model;
+                    this.listenTo(collection, 'sync', function() {
+                        if (collection.searchString) {
+                            headerModel.set(
+                                'title',
+                                interpolate(
+                                    gettext('Search Topic: %(topic)s'),
+                                    { topic: subject.get('name') },
+                                    true
+                                )
+                            );
+                            headerModel.set(
+                                'description',
+                                interpolate(
+                                    gettext('Matching "%(searchString)s"'),
```

Comment:
This string doesn't seem to be getting picked up for internationalization, though that might just be a sandbox issue.